frontend/src/components/Button/index.js

```diff
@@ -1,12 +1,13 @@
 import { PropTypes } from "prop-types";
 import styles from "./index.module.css";
 
-const Button = (props) => {
-  return <button className={styles.button}>{props.children}</button>;
-};
+const Button = ({children, onClick}) => {
+  return <button className={styles.button} onClick={onClick}>{children}</button>;
```

Comment:
You don't have an `onClick` function defined.

Reply:
You can remove it for now.

Reply:
I'm surprised your linter didn't complain? Can you check if it's set up correctly before you make the change?

Reply:
Okay I'll remove it. I put it because since you said onClick was a prop I thought I could pass a handler inside this button and it would be useful in the future :sweat_smile:

Reply:
It's a prop and we'll need it but at the moment you don't have any functionality. There's no spec on what to do `onClick.` We'll get to it once we start properly using the `Button` component. Also, some other attributes are missing, so we're going to need to check the Lint settings, as Justo mentioned.